ansible/roles/datadog/tasks/main.yml

```diff
@@ -38,5 +38,6 @@
     accept_hostkey: yes
     update: yes
   sudo_user: "{{ cchq_user }}"
+  notify: restart datadog
```

Comment:
Does this always restart datadog, or only restarts it when there are new commits pulled?